docs/getting_started/01-prerequisites.md

```diff
@@ -1,62 +1,57 @@
 ---
-id: getting-started-onboarding
-slug: /getting-started/onboarding
-title: Onboarding
-sidebar_label: Onboarding
+id: getting-started-prerequisites
+slug: /getting-started/prerequisites
```

Comment:
Looks cool! Be sure to add a redirect from /getting-started/onboarding to /getting-started/prerequisites so google seo console does bzzt me.

Reply:
Added, and tested redirect